examples/subplots_axes_and_figures/subplot.py

```diff
@@ -15,14 +15,14 @@
 y1 = np.cos(2 * np.pi * x1) * np.exp(-x1)
 y2 = np.cos(2 * np.pi * x2)
 
-plt.subplot(2, 1, 1)
```

Comment:
From the filename, I'm not sure if this file is actually _about_ `subplot`...

Reply:
Did you mean it should be an example of the subplot() function? The page is referenced from the section “ Multiple Subplots”, so I guess it does not specifically refer to subplot().

Reply:
I think its reasonable to have what we feel is the canonical way here.  

Can I suggest you add a second figure that shows `subplot` though with a bit of preceding text that says it is an alternate way using `pyplot`?

Reply:
BTW, the rest seems fine.  Ping for a review, and thanks for working on this!

Reply:
Not sure I fully understand what you mean, you recommand adding a second example in the same page with the original method of `subplot`?

Reply:
Yes.  Each example can have multiple figures on it.